types/chance/index.d.ts

```diff
@@ -28,16 +29,16 @@ declare namespace Chance {
         bool(opts?: Options): boolean;
         character(opts?: Options): string;
         floating(opts?: Options): number;
-        integer(opts?: Options): number;
+        integer(opts?: {min: number, max: number}): number;
```

Comment:
my reading of this is that, as you've typed it, you can't provide a min _without_ a max, which isn't the case. (the doc doesn't indicate it, but if you look at the source code or run it yourself, you'll see that you _can_ provide one without the other.)

you'll need to make the min/max optional like the other functions have.

```suggestion
        integer(opts?: {min?: number, max?: number}): number;
```